cime_config/SystemTests/rxcropmaturity.py

```diff
@@ -56,11 +62,17 @@ def __init__(self, case):
                 f"STOP_OPTION ({stop_option_orig}) must be nsecond(s), nminute(s), "
                 + "nhour(s), nday(s), nmonth(s), or nyear(s)"
             )
-        elif stop_n < 5:
+        elif full_test and stop_n < 5:
             error_message = (
                 "RXCROPMATURITY must be run for at least 5 years; you requested "
                 + f"{stop_n_orig} {stop_option_orig[1:]}"
             )
+        elif skipgen_test and stop_n < 3:
+            # First year is discarded because crops are already in the ground at restart, and those aren't affected by the new crop calendar inputs. The second year is useable, but we need a third year so that all crops planted in the second year have a chance to finish.
```

Comment:
This comment is really long and has to wrap. I thought pylint wouldn't allow that, but it obviously does. My preference is to keep comments readable lengths without wrapping.

I realize this is a personal preference but my reasoning is that it's easier to tell the whole block is a comment if each wrapped line  has the "#" in front of it. It also keeps the whole comment even if your editor is setup to not wrap long lines. So I would say it helps with readability.

There's a few others like this as well in this code.

The same can be true in other contexts as well (for example XML and there is an example below). I find that more acceptable because there is something that signify's the end.

Reply:
`pylint` isn't set up to check the SystemTests directory, which is why it's allowed (although `black` does).

My personal preference is to keep comments all on one line for two reasons.
1. If you do have line wrapping turned on, it improves readability at smaller screen sizes / larger text sizes (and is thus an accessibility feature). Comments with manual line breaks wrap, then you see a few characters of the next line, and then it breaks.
2. It makes it easier to edit comments. If you need to add text in the middle, you don't have to manually re-break everything again.

Modern editors should make it easy to enable soft wrapping; I believe it's the default for `vim`, and in VSCode it can be set as the default (or given a keyboard shortcut). Modern editors should also include text styling that makes it easy to tell what lines are part of a comment.

All that said, I've manually wrapped the long comments in this file (98e3689e0), so I'll mark this as resolved.